src/baseline_bots/utils.py

```diff
@@ -453,3 +453,18 @@ def smart_select_support_proposals(
             if len(order_list) > 1:
                 optimal_ordering_units.add(ordering_unit)
     return optimal_possible_support_proposals
+
+
+# return a list of powers that are neighbors of power_name
+def neighboring_opps(game, power_name, opponents) -> List[str]:
+    neighbors = set()  # set to prevent duplicates
+    adj_provs = set()  # provs adjacent to power_name territories
+    for prov in game.powers[power_name].influence:
+        adj_provs.update(set(x.upper() for x in game.map.abut_list(prov)))
+
+    for opponent in opponents:
+        for prov in game.powers[opponent].influence:
+            if prov in adj_provs:
+                neighbors.update(opponent)
```

Comment:
You probably wanted the following:

```suggestion
                neighbors.add(opponent)
```

With the current code, you'll get undesired behavior:

```python
>>> s = set()
>>> s.update('AUS')
>>> s
{'S', 'A', 'U'}
>>> t = set()
>>> t.add('AUS')
>>> t
{'AUS'}
```

Reply:
That makes sense. I'll change that.